### ISSUE TITLE
custom .eslintrc gives false negatives for some configurations

### ISSUE THREAD

**Issue by timaschew** [houndci/eslint#27]
Without a __.hound.yml__ file hound is working: https://github.com/timaschew/hound-test/pull/1

When I use this yml file

```yml
javascript:
  enabled: false
eslint:
  enabled: true
  config_file: .eslintrc
```

And use the [default hound config] (https://raw.githubusercontent.com/houndci/eslint/master/config/.eslintrc)
and just change the `es6` value to:

```json
{
    "rules": {},
    "env": {
        "es6": false,
        "browser": true
    },
    "extends": "eslint:recommended"
}
```
Hound works as well: https://github.com/timaschew/hound-test/pull/4

But If I change .eslintrc to

```json
{
  "extends": "eslint:recommended",
  "env": {
    "es6": true,
    "browser": false
  }
}
```
it's not working anymore: https://github.com/timaschew/hound-test/pull/2 
Hound says it didn't found any violations but there are, if I run `npm run lint`
Same effect if I extend the airbnb rules: https://github.com/timaschew/hound-test/pull/3

I'm using eslint 1.10.3

**Comment by timaschew** [comment]
```json
{
    "rules": {},
    "env": {
        "es6": true,
        "browser": true
    },
    "extends": "airbnb"
}
```
Is working, hmm...

**Comment by timaschew** [comment]
solved, see #29 
summary: Hound is not creating comments/showing violations if the master branch contains violations.
